articles/machine-learning/how-to-machine-learning-interpretability-automl.md

```diff
@@ -118,6 +130,18 @@ engineered_explanations = explainer.explain(['local', 'global'], eval_dataset=au
 print(engineered_explanations.get_feature_importance_dict())
 ```
 
+### Use Mimic Explainer for computing and visualizing raw feature importance
+
+You can call the `explain()` method in MimicWrapper with the transformed test samples to get the feature importance for the generated engineered features. You can also use `ExplanationDashboard` to view the dashboard visualization of the feature importance values of the raw features.
```

Comment:
It might be good to link to documentation on the `ExplanationDashboard` here if we're going to reference the object but then not actually show how to use it.

Reply:
Thanks for pointing this out. Actually, we don't have the ExplanationDashboard in the AutoML notebook. So changed it navigate to https://ml.azure.com/ instead.